### PR TITLE
[Merged by Bors] - feat(Algebra/Ring/Semiconj): port file

### PR DESCRIPTION
**Pull request by j-loreaux**
mathlib3 SHA: 76171581280d5b5d1e2d1f4f37e5420357bdc636

porting notes: 
1. Woot! mathport succeeded with no help!
2. I had to remove the `simp` attribute on two declarations because `simpNF` said so. I left porting notes.

### CODE REVIEW

File: Mathlib/Algebra/Ring/Semiconj.lean [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+@[simp]
+theorem sub_left (ha : SemiconjBy a x y) (hb : SemiconjBy b x y) :
+    SemiconjBy (a - b) x y := by
+  simpa only [sub_eq_add_neg] using ha.add_left hb.neg_left
+#align semiconj_by.sub_left SemiconjBy.sub_left
+
+end
+
+end SemiconjBy
+#lint
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Ring/Semiconj.lean:101]:
```suggestion
```

**Review by Ruben-VandeVelde — APPROVED** [review]
Yay!

File: Mathlib/Algebra/Ring/Semiconj.lean (hunk 1 above)
Comment (j-loreaux) [reply Mathlib/Algebra/Ring/Semiconj.lean]: Oops, thanks

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**